packages/merkle-distributor/scripts/example.json

```diff
@@ -0,0 +1,41 @@
+{
+  "chainId": 42,
+  "rewardToken": "0x47B1EE6d02af0AA5082C90Ea1c2c14c70399186c",
+  "windowIndex": 0,
+  "totalRewardsDistributed": "15000000000000000000",
+  "windowStart": 1614850539,
+  "recipients": {
+    "0x00b591bc2b682a0b30dd72bac9406bfa13e5d3cd": {
+      "amount": "1000000000000000000",
+      "metaData": { "reason": ["YD-WETH-21 Liquidity Mining Week 27"] }
+    },
+    "0x00e4846e2971bb2b29cec7c9efc8fa686ae21342": {
+      "amount": "2000000000000000000",
+      "metaData": { "reason": ["YD-WETH-21 Liquidity Mining Week 27", "Governance Votes KPI Options Round 1"] }
+    },
+    "0x00e4f5a158ec094da8cf55f8d994b84b6f5f33d9": {
+      "amount": "3000000000000000000",
+      "metaData": {
+        "reason": [
+          "YD-WETH-21 Liquidity Mining Week 27",
+          "Governance Votes KPI Options Round 1",
+          "Token Minter KPI Options Round 1"
+        ]
+      }
+    },
+    "0x0156ff30a13abbe641cb17f33967eb67feafeb4e": {
+      "amount": "4000000000000000000",
+      "metaData": {
+        "reason": [
+          "YD-WETH-21 Liquidity Mining Week 27",
+          "Governance Votes KPI Options Round 1",
+          "Token Minter KPI Options Round 1"
+        ]
+      }
+    },
+    "0x01b01c6fb158e55c83883219a533f1efd2abfdf4": {
+      "amount": "5000000000000000000",
+      "metaData": { "reason": ["Governance Votes KPI Options Round 1"] }
```

Comment:
Do you set `metaData` in any of the scripts or is this done offline?

Reply:
All information in this file is generated pre-generated from a separate script. This is just an example to show what you can put in `metaData` any arbitrary data type.